src/components/SettingComponent.js

```diff
@@ -148,17 +149,20 @@ export const PopoverColorPicker = ( { name, value, icon, storageKey, onChange, .
 
 PopoverColorPicker.propTypes = {
   value: string.isRequired,
-  icon: any.isRequired,
+  icon: shape( {} ).isRequired,
   name: string.isRequired,
   storageKey: string.isRequired,
   onChange: func.isRequired,
 }
 
```

Comment:
Why do we need a component here for this? Can we not just return `null` at the point we need to render it when there is no supplied option type?

Reply:
This is required for popover, as you would want to click something. but now I realized that I can refactor this because same icon is being passed for color picker.